__tests__/plugins/tracing.test.ts

```diff
@@ -37,19 +37,26 @@ describe('tracing', () => {
     await server.close();
   });
 
-  it('capture filmstrips', async () => {
+  it('capture trace events', async () => {
     const driver = await Gatherer.setupDriver({ wsEndpoint });
-    const tracer = new Tracing(driver, { filmstrips: true });
+    const tracer = new Tracing(driver, { filmstrips: true, trace: true });
     await tracer.start();
-    await driver.page.goto(server.TEST_PAGE);
-    await driver.page.waitForLoadState('load');
-    const { filmstrips } = await tracer.stop();
+    await driver.page.goto(server.TEST_PAGE, { waitUntil: 'networkidle' });
+    const { filmstrips, traces } = await tracer.stop();
     await Gatherer.stop();
-    expect(filmstrips.length).toBeGreaterThan(0);
-    expect(filmstrips[0]).toMatchObject({
-      blob: expect.any(String),
-      mime: 'image/jpeg',
-      start: { us: expect.any(Number) },
-    });
+    if (filmstrips.length > 0) {
```

Comment:
If we're doing this sort of thing we should definitely have a long comment explaining why. It's just encapsulating the flakiness in the test, which is an improvement, but we should do our best to explain before merging

Reply:
Added comment for the same.